src/main/java/booktopiahub/controller/BookController.java

```diff
@@ -28,43 +30,44 @@
 public class BookController {
     private final BookService bookService;
 
-    @PreAuthorize("hasRole('ADMIN')")
+    @PreAuthorize("hasRole('USER') OR hasRole('ADMIN')")
     @GetMapping
     @Operation(summary = "Get all books", description = "Get a list of all available books")
     public List<BookDto> findAll(@ParameterObject Pageable pageable) {
         return bookService.getAll(pageable);
     }
 
-    @PreAuthorize("hasRole('USER')")
+    @PreAuthorize("hasRole('USER') OR hasRole('ADMIN')")
     @GetMapping("/{id}")
     @Operation(summary = "Get book by id", description = "Get available book by id")
     public BookDto getBookById(@PathVariable Long id) {
         return bookService.findById(id);
     }
 
-    @PreAuthorize("hasRole('ADMIN')")
+    @PreAuthorize("hasRole('USER') OR hasRole('ADMIN')")
```

Comment:
Please recheck through project
```suggestion
    @PreAuthorize("hasAnyRole('USER', 'ADMIN')")
```